lesson01/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Freeland - Assignment 1 Part 2</title>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="scripts.js"></script>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <h1>ES6+ Methods</h1>
+
+  <div class="non-h1-body">
+    <b><p>Use these input boxes to find out information about the following array of numbers:</p></b>
+    <div id="array"></div>
+
+    <p>----------</p>
```

Comment:
Consider using the horizontal rule element here to achieve a similar visual effect in a more semantically accurate way: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr

Reply:
Oh, that looks so much better. Thank you!